zipkin-collector/activemq/README.md

```diff
@@ -0,0 +1,56 @@
+# collector-activemq
+
+## ActiveMQCollector
+This collector consumes a ActiveMQ queue for messages that contain a list of spans.
+Its only dependencies besides Zipkin core are the `slf4j-api` and the [ActiveMQ Java Client](https://github.com/apache/activemq).
+
+### Configuration
+
+The following configuration can be set for the ActiveMQ Collector.
+
+Property | Environment Variable | Description
+--- | --- | ---
+`zipkin.collector.activemq.connection-timeout` | `ACTIVE_CONNECTION_TIMEOUT` | Milliseconds to wait establishing a connection. Defaults to `60000` (1 minute)
+`zipkin.collector.activemq.queue` | `ACTIVE_QUEUE` | Queue from which to collect span messages. Defaults to `zipkin`
+
+If the URI is set, the following properties will be ignored.
+
+Property | Environment Variable | Description
+--- | --- | ---
+`zipkin.collector.activemq.addresses` | `ACTIVE_ADDRESSES` | Comma-separated list of ActiveMQ addresses, ex. `tcp://localhost:61616,tcp://localhost2:61616`
```

Comment:
is the naming convention `ACTIVE` normal in activemq or is it `ACTIVEMQ` or different? can you give example of some commandline tool which uses this convention?